### ISSUE TITLE
*/ inside code block

### ISSUE THREAD

**Issue by Ignacja** [doxygen/doxygen#6737]
I want to put inside a page yaml code with line:
```
        projects: '**/*.test/*.csproj'
```
and I have the warning:
```
$ warning: found */ without matching /* while inside a \code block! Perhaps a missing \endcode?
```
How can I escape the "*/" so there will be no warnings?

**Comment by albert-github** [comment]
To be able to see what is the problem some more information is needed and yaml files are not directly supported I assume you use a filter, its results are quite important as well.

In short: Can you please attach a, small, self contained example (source+config file in a tar or zip) that allows us to reproduce the problem?
Please also specify the doxygen version used.

**Comment by bourtemb** [comment]
Hi,

We get the same warning with doxygen 1.8.13 when trying to add a C++ code sample in the Doxygen documentation:
```
/**
 * Removes any group or device which name matches the specified pattern.
 * @code
 * root_group->remove("*/stepper_motor/*");
 * @endcode
 *
 * @param [in] pattern The device selection pattern
 */
  virtual void remove (const std::string& pattern);
```

**Comment by bourtemb** [comment]
@albert-github, indeed, in my case, the comment block is protected with something like 
#if DOXYGEN_ONLY

Sorry, I should have put that in the code sample I shared...

**Comment by albert-github** [comment]
Part of the problem was solved already in the 1.8.16 version, but there was still a remaining problem with some left over `*` at the beginning of the line, for this I've just pushed a proposed patch, pull request #9868

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.7.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
